kitsune/sumo/middleware.py

```diff
@@ -103,29 +92,21 @@ def process_request(self, request):
         expiration = request.session.get("oidc_id_token_expiration", 0)
         now = time.time()
         access_token = request.session.get("oidc_access_token")
```

Comment:
We don't actually do anything with the access_token, so I wonder if there's any point in storing this in the session?
Similarly the `oidc_id_token_expiration` value is something we're setting ourselves.

So I wonder, could we just use something like a `session["fxa_session_expiry"]` value, and bump that value by `FXA_RENEW_SESSION_EXPIRY_SECONDS` when using the refresh_token works (but then immediately discarding the access_token).

Though, if we wanted to (now or in the future) hit the verify endpoint with the access_token, before attempting a refresh when that fails, there's value in keeping the access_token in the session. In that case - we probably don't want to send a `ttl` argument in our refresh request, to allow FxA to dictate how often we hit the more expensive refresh route.

Reply:
I am updating the PR to exclude `ttl` from here. It will be an optional param used by the `process_event_profile_change` task